css-cascade/specificity.css

```diff
@@ -0,0 +1,19 @@
+* {
```

Comment:
These selectors should be in the reverse order to demonstrate that source order will be overridden by a more specific selector

Reply:
gotcha, redid that section of the exercise